Website/Packages/datax-pipeline/src/modules/flowDefinition/components/schedule/scheduleSettingsContent.jsx

```diff
@@ -0,0 +1,329 @@
+// *********************************************************************
+// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License
+// *********************************************************************
+import React from 'react';
+import PropTypes from 'prop-types';
+import * as Models from '../../flowModels';
+import { DetailsList, DetailsListLayoutMode, Selection, SelectionMode, CheckboxVisibility, DefaultButton } from 'office-ui-fabric-react';
+import { Colors, IconButtonStyles, PanelHeader, PanelHeaderButtons, ScrollableContentPane, StatementBox } from 'datax-common';
+import RecurringScheduleSettings from './recurringScheduleSettings';
+import OneTimeScheduleSettings from './oneTimeScheduleSettings';
+import * as Styles from '../../../../common/styles';
+
+const batchColumns = [
+    {
+        key: 'columnBatchName',
+        name: 'Alias',
+        fieldName: 'id',
+        isResizable: true
+    },
+    {
+        key: 'columnBatchType',
```

Comment:
please look at the office-ui-fabric documentations for the DetailsList control, I recommend the name column take 50% of the space and Type and Status columns each take 25%

Reply:
I made this look better by setting the miWidth for typeDisplay and Status columns.